app/Models/DatabaseDAOPGSQL.php

```diff
@@ -58,12 +58,12 @@ public function size($all = false) {
 			$stm->execute();
 		} else {
 			$sql = "SELECT "
-			     . "pg_total_relation_size('`{$this->pdo->prefix()}category`') + "
-			     . "pg_total_relation_size('`{$this->pdo->prefix()}feed`') + "
-			     . "pg_total_relation_size('`{$this->pdo->prefix()}entry`') + "
-			     . "pg_total_relation_size('`{$this->pdo->prefix()}entrytmp`') + "
-			     . "pg_total_relation_size('`{$this->pdo->prefix()}tag`') + "
-			     . "pg_total_relation_size('`{$this->pdo->prefix()}entrytag`')";
+				. "pg_total_relation_size('`{$this->pdo->prefix()}category`') + "
+				. "pg_total_relation_size('`{$this->pdo->prefix()}feed`') + "
+				. "pg_total_relation_size('`{$this->pdo->prefix()}entry`') + "
+				. "pg_total_relation_size('`{$this->pdo->prefix()}entrytmp`') + "
+				. "pg_total_relation_size('`{$this->pdo->prefix()}tag`') + "
+				. "pg_total_relation_size('`{$this->pdo->prefix()}entrytag`')";
```

Comment:
Maybe using the HEREDOC notation here would be useful since you already use the string templates.

Reply:
Indeed, but let's continue addressing them in another PR, not to mix too many things :-)

Reply:
Will do here after all :-)

Reply:
https://github.com/FreshRSS/FreshRSS/pull/3488/commits/c86067c0de096057fa1e494d5fa947e93e09b81f